test/app.js

```diff
@@ -22,15 +22,42 @@ var helpers = require('yeoman-test');
 var basicProps = {};
 var fs = require('fs-extra');
 const utils = require('../app/util');
+const app_soap = require('./app_soap');
+
+
+var defaultCamel = '2.22.2';
 
 describe('generator-camel:app', function () {
 
+  describe('Should test the utils class package validation', function () {
+    it('utilities package validation should work for valid package', function () {
+      assert.strictEqual(utils.validatePackage('com.valid'), true);
+    });
+    it('utilities package validation should fail for package name with invalid characters', function () {
+      assert.notStrictEqual(utils.validatePackage('invalid@.pkg.name'), true);
+    });
+    it('utilities package validation should fail for package name with java keyword', function () {
+      assert.notStrictEqual(utils.validatePackage('a.name.with.package'), true);
+    });
+
+    it('utilities findWsdl2RestJar should succeed in finding wsdl2rest jar', function () {
+      // test runs on its own but fails in the larger test suite - still figuring that out
```

Comment:
is it still true? there is no error on PR CI job

Reply:
Not true - will remove PR